linux/VSTPlugin-linux.cpp

```diff
@@ -23,20 +23,49 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 AEffect* VSTPlugin::loadEffect() {
 	AEffect *plugin = nullptr;
 
-	wchar_t *wpath;
-	os_utf8_to_wcs_ptr(pluginPath.c_str(), 0, &wpath);
-	soHandle = LoadLibraryW(wpath);
+	// Why is this a (non-portable) wide char?
```

Comment:
That's copied from the windows code. Windows needs a wchar because the filesystem APIs for unicode accept a wchar. On linux and osx you should just be able to pass the string to the os_dlopen. So you can remove all this extra stuff here.